docs-partials/builder/chroot/Config-not-required.mdx

```diff
@@ -159,6 +159,10 @@
   more information. Defaults to `legacy-bios` when `ami_architecture` is `x86_64` and
   `uefi` when `ami_architecture` is `arm64`.
 
+- `tpm_support` (string) - The TPM Support mode. Valid options are `v2.0`. See the documentation on
```

Comment:
The doc snippet looks good to me, for completeness I'd suggest adding this to all the builders that support the feature, so in addition to the `chroot` builder, the `ebssurrogate` and the `instance` builder docs should add this to their sections.